### PR TITLE
feat: add aliases to improve ux

### PR DESCRIPTION
**Pull request by jrschumacher**
Add aliases to make UX better

### CODE REVIEW

File: docs/man/policy/subject-condition-sets/_index.md [hunk 1]
```diff
@@ -2,4 +2,7 @@
 title: Subject condition sets
 command:
   name: subject-condition-sets
+  aliases:
+    - subcs
```
Comment (jakedoublev) [inline docs/man/policy/subject-condition-sets/_index.md:6]:
```suggestion
    - subcs
    - scs
```

File: docs/man/policy/subject-mappings/_index.md [hunk 2]
```diff
@@ -2,4 +2,7 @@
 title: Subject mappings
 command:
   name: subject-mappings
+  aliases:
+    - subm
```
Comment (jakedoublev) [inline docs/man/policy/subject-mappings/_index.md:6]:
```suggestion
    - subm
    - sm
```

File: docs/man/policy/resource-mappings/_index.md [hunk 3]
```diff
@@ -2,6 +2,9 @@
 title: Manage resource mappings
 command:
   name: resource-mappings
+  aliases:
+    - resm
```
Comment (jakedoublev) [inline docs/man/policy/resource-mappings/_index.md:6]:
```suggestion
    - resm
    - remap
```

**Comment by jakedoublev** [comment]
Discussed these verb aliases as well:

- list as `ls`
- delete as `rm`
- create as `new`

**Comment by jakedoublev** [comment]
Looks like we have `ls` and `l` both for LIST of namespaces, attributes, and values, but only `l` for `kas-registry`, `subject-mappings`, `subject-condition-sets`, and `resource-mappings`. Is that desired behavior?

File: docs/man/policy/attributes/list.md [hunk 4]
```diff
@@ -2,6 +2,9 @@
 title: List attributes
 command:
   name: list
+  aliases:
+    - ls
+    - l
```
Comment (jakedoublev) [inline docs/man/policy/attributes/list.md:7]: Looks like we have ls and l both for LIST of namespaces, attributes, and values, but only l for kas-registry, subject-mappings, subject-condition-sets, and resource-mappings. Is that desired behavior?

**Review by jakedoublev — APPROVED** [review]

File: docs/man/policy/attributes/list.md [hunk 5]
```diff
@@ -2,6 +2,9 @@
 title: List attributes
 command:
   name: list
+  aliases:
+    - ls
```
Comment (jrschumacher) [inline docs/man/policy/attributes/list.md:6]:
```suggestion
```

Comment (jrschumacher) [inline docs/man/policy/attributes/list.md]:
```suggestion
```

**Review by jakedoublev — APPROVED** [review]

**Pull request merged**